ray-operator/config/samples/ray-cluster.heterogeneous.yaml

```diff
@@ -173,7 +171,7 @@ spec:
         initContainers:
         - name: init-myservice
           image: busybox:1.28
-          command: ['sh', '-c', "until nslookup raycluster-heterogeneous-my-svc.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local; do echo waiting for myservice; sleep 2; done"]   
+          command: ['sh', '-c', "until nslookup raycluster-heterogeneous-head-svc.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local; do echo waiting for myservice; sleep 2; done"]
```

Comment:
the default svc name was not `raycluster-heterogeneous-my-svc`, It was `raycluster-heterogeneous-head-svc`.
So there was an error as follows:
![hetero-medium-svc-name-wrong](https://user-images.githubusercontent.com/37469330/135426024-2c0cabe6-8f94-4308-ad42-b24255b65a9b.png)

![hetero-default-svc-name](https://user-images.githubusercontent.com/37469330/135426036-b636a6d0-2aeb-43e4-adfb-3078fa176f06.png)

Reply:
This is a nice catch!  Thanks!